client/ayon_traypublisher/plugins/create/create_editorial.py

```diff
@@ -843,7 +843,12 @@ def get_pre_create_attr_defs(self):
                 ],
                 allow_sequences=False,
                 single_item=False,
-                label="Sequence file",
+                label="Edit list file",
```

Comment:
**Edit List** file - that's not actually a thing right, because it's missing the **Decision** word in there?

Should this be any  of:
```suggestion
                label="Edit Decision List file",
```

```suggestion
                label="EDL file",
```

```suggestion
                label="Edit Decision List",
```

Reply:
I did quick research and found out that the original name `Sequence file`  is originated from Nuke feature [Exporting Sequences as EDL, OTIO, and XML](https://learn.foundry.com/nuke/content/timeline_environment/exporting/exporting_edls_and_xmls.html)

Personally, I find `Sequence file` but does calling `Edit Decision List`  is accurate? because it supports other formats.

Reply:
Tagging @jakubjezek001 and @robin-ynput for visibility.

Reply:
So I belive `Edit Decision List` is the correct term here, with (.edl, .xml, .fcpxml and .aaf being the EDL formats that are supported). In term of what kind of information it contains, some DCC will call it a `Sequence` others a `Timeline` but this is usually the same thing.